src/Service/FeedManager.php

```diff
@@ -180,4 +183,70 @@ public function getFeedList(): array
 
         return $feedsResponse;
     }
+
+    public function getFeedCount(): FeedCount
+    {
+        $action = 'FeedCount';
+        $requestId = uniqid((string) mt_rand());
```

Comment:
maybe gerenate the id like this?
```php
    return bin2hex(random_bytes(16));
```

Reply:
yes, we decided that today morning. Thanks